### ISSUE TITLE
CAMPAIGN: Color contrast does not meet minimum requirement

### ISSUE THREAD

**Issue by poojagunturu96** [middlebury/midd-frontend#712]
Siteimprove report - https://my2.siteimprove.com/Accessibility/7333/NextGen/Issue/1?conformance=&siteTargetIssueKinds=1,2&wcagVersion=21&pageSegments=&ruleName=sia-r69&ruleId=69&issueKind=1&tagFilterType=2&siteTarget.wcagVersion=21&siteTarget.conformanceLevels=0,1,3,4&siteTarget.issueKinds=1,2

**Comment by poojagunturu96** [comment]
Resolved in https://github.com/middlebury/public-campaign-site/commit/62a49006a959f5deee3a8dd79b3f3dd1835a8f34.
